### PR TITLE
mysql fix

### PR DESCRIPTION
**Pull request by braddevans**
changed jdbc connection url 

config addition:
```yaml
Mysql:
  Host: "localhost"
  Port: "3306"
  Database: "bungeeperms"
  User: "permsuser"
  Password: "password"
```

### CODE REVIEW

**Comment by braddevans** [comment]
still working on the fix

File: src/main/java/net/alpenblock/bungeeperms/BungeePerms.java [hunk 1]
```diff
@@ -7,6 +7,7 @@
 import net.alpenblock.bungeeperms.platform.NetworkNotifier;
 import net.alpenblock.bungeeperms.platform.PlatformPlugin;
 import net.alpenblock.bungeeperms.platform.PluginMessageSender;
+import org.bukkit.configuration.file.FileConfiguration;
```
Comment (weaondara) [inline src/main/java/net/alpenblock/bungeeperms/BungeePerms.java]: this import is not working in this class ... the BungeePerms class is intended to not depend on bukkit or bungee api

File: src/main/java/net/alpenblock/bungeeperms/BungeePerms.java [hunk 2]
```diff
@@ -110,4 +112,12 @@ public void reload(boolean notifynetwork)
         }
         enable();
     }
+
+    public static BungeePerms getInstance() {
```
Comment (weaondara) [inline src/main/java/net/alpenblock/bungeeperms/BungeePerms.java]: this method is already generated by lombok

File: src/main/java/net/alpenblock/bungeeperms/BungeePerms.java [hunk 3]
```diff
@@ -110,4 +112,12 @@ public void reload(boolean notifynetwork)
         }
         enable();
     }
+
+    public static BungeePerms getInstance() {
+        return BungeePerms.instance;
+    }
+
+    public FileConfiguration getExtraConfig() {
```
Comment (weaondara) [inline src/main/java/net/alpenblock/bungeeperms/BungeePerms.java]: this method is already generated by lombo

**Comment by weaondara** [comment]
what exactly does it pr fix

**Comment by braddevans** [comment]
i was working on the database initial connection
which doesn't work

**Comment by weaondara** [comment]
it doesn't?

**Comment by braddevans** [comment]
nope i tried several users i created with all simple passwords and hard passwords none of them worked even with testing all different versions of mysql and mariadb

**Comment by aurorasmiles** [comment]
You're doing something wrong then. Tested and works with MySQL as well as MariaDB. Please open an issue with more information, instead of trying to fix an issue that doesn't exist.

**Pull request closed without merging**